### PR TITLE
fixes the issue where process=all should default to process=[parent, content, gpu]

### LINKED ISSUE #417: handle firefox desktop probes w/ the "all" process
There are a few cases where we have `all` listed in `probe.record_in_processes`.

The work to close this issue is (1) find out what "all" means in this contexts, and (2) do something either on the probe API side or the client side to transform "all" to something more useful (like listing the processes that are used). I would opt for this to be fixed on the server end, so the data is cleaner going into the frontend.

Here is the list:

```
keypress_present_latency
telemetry_event_recording_error
uptake_remote_content_result_1
browser_usage_graphite
telemetry_event_counts
telemetry_profile_directory_scans
telemetry_accumulate_unknown_histogram_keys
telemetry_keyed_scalars_unknown_keys
```

### PR DESCRIPTION
**Pull request by hamilton**
Closes #417 

We have some high-value probes that mysteriously have `process=all` such as `keypress_present_latency`. This fixes the case by creating a new `transformProbeForGLAM` function within each config file, which does what you would expect. I am imagining that we'll likely encounter issues that require this kind of function on a per-product basis.

### CODE REVIEW

File: src/config/firefox-desktop.js [hunk 1]
```diff
@@ -43,6 +43,13 @@ export default {
       },
     },
   },
+  transformProbeForGLAM(probe) {
```
Comment (spasovski) [inline src/config/firefox-desktop.js:46]: This is a good solution but I still wish it wasn't necessary.

**Review by spasovski — APPROVED** [review]
Looks good thanks!

File: src/config/firefox-desktop.js (hunk 1 above)
Comment (robhudson) [reply src/config/firefox-desktop.js:46]: Perhaps that's a change that could be pushed to probe-scraper, to ultimately end up in probe-info?

Comment (hamilton) [reply src/config/firefox-desktop.js:46]: I think that would be ideal. Do you know how to go about requesting that @robhudson? Is the probe scraper versioned?

Comment (robhudson) [reply src/config/firefox-desktop.js:46]: By the way, there's also `"all_childs"`: https://firefox-source-docs.mozilla.org/toolkit/components/telemetry/collection/histograms.html#record-in-processes

Comment (robhudson) [reply src/config/firefox-desktop.js:46]: `"all_childs"` -> `['content', 'gpu']` (all except `'main'`)

Comment (robhudson) [reply src/config/firefox-desktop.js:46]: https://github.com/mozilla/probe-scraper/issues/187

**Pull request merged**